### ISSUE TITLE
Documenting issues in one big list for you

### ISSUE THREAD

**Issue by LittleWishy** [hfellerhoff/openipa#543]
[OpenIPA French Issues.pdf](https://github.com/user-attachments/files/17088300/OpenIPA.French.Issues.pdf)
Here is a list that I have made of functions that need to be added, removed, and considered for OpenIPA French. This is probably an unfinished list, as it is likely that I missed some things. Hopefully organizing them in this format can help you understand better how to fix or implement certain features. Feel free to let me know if you have questions.

**Comment by hfellerhoff** [comment]
Wow - this is so well put together, thank you for this! Are you good with me reaching out if I need clarification as I work through these?

**Comment by LittleWishy** [comment]
@hfellerhoff Of course! Feel free to ask me anything, I'll do what I can to give you my best answers.
